test/promiseAllProperties.spec.ts

```diff
@@ -8,13 +8,26 @@ const expect = chai.expect;
 describe('promiseAllProperties', () => {
   describe('validate rejections', () => {
     it('should reject if not receiving an object', () => {
-      return Promise.all([
-      expect((promiseAllProperties as any)(/*undefined*/)).to.be.rejectedWith(TypeError, 'The input argument must be of type Object'),
-        expect((promiseAllProperties as any)(null)).to.be.rejectedWith(TypeError, 'The input argument must be of type Object'),
-        expect((promiseAllProperties as any)('string')).to.be.rejectedWith(TypeError, 'The input argument must be of type Object'),
-        expect((promiseAllProperties as any)(123)).to.be.rejectedWith(TypeError, 'The input argument must be of type Object'),
-        expect((promiseAllProperties as any)(true)).to.be.rejectedWith(TypeError, 'The input argument must be of type Object'),
-      ]);
+      const promises = [
+        // @ts-expect-error  (testing bad input)
+        promiseAllProperties(/*undefined*/),
+        // @ts-expect-error  (testing bad input)
+        promiseAllProperties(null),
+        // @ts-expect-error  (testing bad input)
+        promiseAllProperties('string'),
+        // @ts-expect-error  (testing bad input)
+        promiseAllProperties(123),
+        // @ts-expect-error  (testing bad input)
+        promiseAllProperties(true),
+      ].map((promise) =>
```

Comment:
```suggestion
        // @ts-expect-error  (testing bad input)
        promiseAllProperties([Promise.resolve(1)]),
      ].map((promise) =>
```

Here we'd add passing an Array as something that should throw/reject